src/component/geo/GeoView.js

```diff
@@ -39,12 +39,17 @@ export default echarts.extendComponentView({
             return;
         }
 
-        var mapDraw = this._mapDraw;
         if (geoModel.get('show')) {
-            mapDraw.draw(geoModel, ecModel, api, this, payload);
+            if (this._mapDraw.hasDraw) {
+                this._mapDraw.group.show();
+            }
+            else {
+                this._mapDraw.draw(geoModel, ecModel, api, this, payload);
```

Comment:
We need to call the `draw` every time to update the map. In this way, it's only called the first time.

We can fix this issue by adding the groups back every time in the code https://github.com/apache/incubator-echarts/blob/master/src/component/helper/MapDraw.js#L152 . Now it is only been added on init.

Reply:
That means we should call `new MapDraw` on every update?

Reply:
Oh yeah, That make sense, because it was removed, so we need to create a new one